rmf_task_msgs/msg/TaskSummary.msg

```diff
@@ -0,0 +1,8 @@
+string task_id
+
+# a brief summary of the current status of the task
+string status
+
+builtin_interfaces/Time submission_time
+builtin_interfaces/Time start_time
+builtin_interfaces/Time end_time
```

Comment:
For tasks that are in progress, should this message be sent with an estimate of the end time?

Or is this message only meant for tasks that have been completed?

Reply:
Considering the `status` field, I suppose this is probably for in-progress tasks as well, so I guess we should give an end time estimate.